### PR TITLE
add Padowetz

### PR DESCRIPTION
**Pull request by iKoulee**
Prvni pokus

### CODE REVIEW

File: restaurants/padowetz [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# vim: set ts=2 sw=2 et fileencoding=utf-8:
+
+import urllib2
+import datetime
```
Comment (StLuke) [inline restaurants/padowetz]: not used import

File: restaurants/padowetz [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# vim: set ts=2 sw=2 et fileencoding=utf-8:
```
Comment (StLuke) [inline restaurants/padowetz:2]: indentation should be multiple of 4 - PEP8 FTW

File: restaurants/padowetz [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import urllib2
+import datetime
+import re
+try: 
+  from BeautifulSoup import BeautifulSoup
+except ImportError:
+  from bs4 import BeautifulSoup
+
+
+response = urllib2.urlopen('http://www.restaurant-padowetz.cz/poledni-menu.htm')
```
Comment (StLuke) [inline restaurants/padowetz]: too long line :)

File: restaurants/padowetz [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+for line in menu.find('table', attrs={'class':'jidelnicek', 'id':'t_Polevky'}):
+  for cell in line:
+    if re.match(r'^nazev[0-9]+.*', cell.attrs[0][1]):
+        print(cell.text)
+
+print 'Hlavní chod:'
+for line in menu.find('table', attrs={'class':'jidelnicek', 'id':'t_Hlavni-chod'}):
+  for cell in line:
+    if re.match(r'^nazev[0-9]+.*', cell.attrs[0][1]):
+      nazev =  cell.text
```
Comment (StLuke) [inline restaurants/padowetz]: two spaces after =

**Review by StLuke — CHANGES_REQUESTED** [review]
I am really really bored :)

File: restaurants/padowetz [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    if re.match(r'^cena[0-9]+.*', cell.attrs[0][1]):
+      cena = cell.text
+  print('{} - {}'.format(nazev.encode('utf-8'), cena.encode('utf-8')))
```
Comment (mrc0mmand) [inline restaurants/padowetz]: Inconsistent parenthesis with other print calls (L17, L23)

**Review by mrc0mmand — CHANGES_REQUESTED** [review]

**Comment by blaskovic** [comment]
2 points from me..

1. Script itself should print restaurant name, in case more restaurants are requested at the same time
2. We do not want note about 'alergens' in the output. Output itself is pretty long anyway so dont make it longer.

```
$ ./padowetz 
Polévky:
Sedlácká polévka (1,3,7,9)
Kuřecí vývar s masem a nudlemi (9)
Hlavní chod:
Srbské rizoto sypané sýrem, okurek (1,7,10) - 99Kč
Kuře na paprice, houskový knedlík (1,3,7) - 99Kč
Original Twister "PADOWETZ" (1,3,7) - 109Kč
Variace listových salátů s rukolou a cherry rajčaty, kuřecí nudličky v corn flakes, bylinkový dip, rozpečená bageta (1,3,7) - 109Kč
Anglický ROASTBEEF propečený do růžova, steakové hranolky, tatarská omáčka (3,10) - 159Kč
!!!! Seznam alergenů je na vyžádání u obsluhy !!!! - -
```

File: restaurants/padowetz (hunk 2 above)
Comment (iKoulee) [reply restaurants/padowetz:2]: I don't care.

**Review by mrc0mmand — APPROVED** [review]

**Pull request merged**